pkg/component/observability/monitoring/prometheus/aggregate/prometheusrules.go

```diff
@@ -67,6 +67,20 @@ func CentralPrometheusRules() []*monitoringv1.PrometheusRule {
 								"summary":     "A node is not healthy.",
 							},
 						},
+						{
+							Alert: "TooManyEtcdSnapshotCompactionJobsFailing",
+							Expr:  intstr.FromString(`count(increase(etcddruid_compaction_jobs_total{succeeded="false"}[3h]) >= 1) / count(increase(etcddruid_compaction_jobs_total[3h]) >= 1) > 0.1`),
+							For:   ptr.To(monitoringv1.Duration("10m")),
+							Labels: map[string]string{
+								"severity":   "warning",
+								"type":       "seed",
+								"visibility": "operator",
+							},
+							Annotations: map[string]string{
+								"description": "Seed {{$labels.seed}} in landscape {{$externalLabels.landscape}} has too many etcd snapshot compaction jobs failing in the past 3 hours.",
```

Comment:
I checked the labels available in the aggregate Prometheus configuration and in the seed Alertmanager when the alert is fired and I found the `seed` label is configured as external label and that the `landscape` label is unfortunately only available in the garden Prometheus but not here. That means, you probably need to drop it for now and change the description to something like the following:

```
Seed {{$externalLabels.seed}} has too many etcd snapshot compaction jobs failing in the past 3 hours.
```

Reply:
Thanks for the suggestion, will go ahead and make this change!

Reply:
Adressed in [1fac9a2](https://github.com/gardener/gardener/pull/9739/commits/1fac9a28b83c3dc7bffd27cce5617e37e1cea461).